### ISSUE TITLE
Archive this project if you no longer want to mantain it

### ISSUE THREAD

**Issue by AleixMT** [jimzrt/Incognito_RCM#45]
Usually this is the first repository that comes up when searching for the incognito homebrew switch app on the Internet. This repository has not got an update since three years ago.

If you are not willing to continue with the project just archive this repository and inform of this decision on the `README.md`. If not you are just causing confusion. And eventually someone could brick his console using your legacy app.

**Comment by Masamune3210** [comment]
Well, if it's not being maintained, it has no maintainer. If there is no maintainer, who exactly is supposed to mark it as decrepit lol

**Comment by AleixMT** [comment]
The author lol who else could it be? He is still contributing to GitHub, but it seems that he is ignoring or not paying attention to this repository:

![image](https://github.com/jimzrt/Incognito_RCM/assets/23342150/e966abd1-82ea-4699-ae8e-a66b12fb2796)

I am trying to solve a problem here. If you know something that I do not tell me.

**Comment by ameameam3** [comment]
agreed, this should be archived regardless of whether or not it's "brick ready"

**Comment by AleixMT** [comment]
@ameameam3 @jimzrt  It is kind of annoying having matches of your homebrew searches with broken apps... Just inform of its state, do not waste people time!

**Comment by jimzrt** [comment]
why do you think you can make any demands on a project that I created in my free time and was nice enough to share the code of? There is no obligation for me to do anything at all and if you had asked nicely I would have done it.
So to be a bit petty - either pay me or ask really really nicely.
Don't waste people's time, just pay up! Easy.

Also Incognito (and all its forks) is long obsolete since atmosphere has this functionality built in now.
If you had simply informed yourself properly, you wouldn't have ended up here in the first place.

**Comment by AleixMT** [comment]
I can make any demand in any public repository that has Issues tab active. It is a different thing that you do not want to fulfill them. 

This is GitHub, this is a professional social network. I do not need to brown-nose you in the issues that I submit into your repository in order for you to address a reasonable non-time-consuming request. If you do not want to have new issues submitted in your own repositories, I recommend that you archive the repository if the project is no longer working or deactivate the Issues tab. 

I asked nicely. At least I do not see that I said anything rude to you. Just pointing out a fact that has an easy and fast solution in your hands. It is for the common good. 

AFAIK Atmosphere does not have the Incognito functionality built-in. Probably you are referring to DNS MITM (?). Correct me if I am wrong:

[Source](https://www.reddit.com/r/SwitchPirates/comments/w1r5kq/help_with_incognito_on_atmosphere_to_safely_use/)
![image](https://github.com/jimzrt/Incognito_RCM/assets/23342150/3151db6f-0631-499f-92ed-79fbbcf4ddf6)

I made some searches to make it sure. 

So, in the past I ended here because from what I saw in your repository the app seemed to be working. 

You do a project on your free time and you prefer to see it broken and people submitting issues complaining of its broken state before informing of it and / or archiving the project considering that this will take less than five minutes to do so? Definetely, we are not on the same page...

So, I guess I will be closing this Issue with a "won't fix" since it seems that you will not archive it, right?

**Comment by jimzrt** [comment]
Sure you can, you can do anything you want.
There is a big difference between brown-nosing, requesting and demanding something.
To say "Just inform of its state, do not waste people time!" comes across as rude to me.

This is github. 
You don't have to use it as "a professional social network".
There are no strings attached at all.
I just decided in the past to share my code.

Still, I wouldn't go out of my way to confuse people or be malicious - I simply do not get notifications for a years old project and only saw this thread because of the @ mention.

The person in the reddit thread is wrong.
exosphere did incorporate the functionality of Incognito: [https://github.com/Atmosphere-NX/Atmosphere/blob/master/docs/features/configurations.md#blanking-prodinfo](https://github.com/Atmosphere-NX/Atmosphere/blob/master/docs/features/configurations.md#blanking-prodinfo)
Incognito doesn't do anything other than blanking prodinfo.

I will archive the project.
All the same, I would work on how to go about voicing such an issue. But you do you.
